sources/engine/Stride.Rendering/Rendering/Images/Fog/FogEffect.sdsl

```diff
@@ -0,0 +1,36 @@
+namespace Stride.Rendering.Images
+{
+    /// <summary>
+    /// Simple fog
+    /// </summary>
+    internal shader FogEffect : ImageEffectShader
+    {
+        stage float FogStart;
+        stage float Density;
+        stage float zFar;
+        stage float zNear;
+        stage bool skipBG;
+
+        [Color]
+        stage float3 FogColor;
+        stage Texture2D DepthTexture;
+
+        stage override float4 Shading() 
+        {
+            float4 color = Texture0.Sample(PointSampler, streams.TexCoord);
+            float z_b = DepthTexture.SampleLevel(PointSampler, streams.TexCoord, 0.0).x;
+            
+            if (!skipBG || z_b < 1.0) {
```

Comment:
Mix of inline and next line braces

Reply:
Resolving this, because it's the same as in the main branch.